packages/core/src/stylable-processor.ts

```diff
@@ -181,33 +199,37 @@ export class StylableProcessor implements FeatureContext {
                     CSSKeyframes.hooks.analyzeAtRule({
                         context: this,
                         atRule,
-                        toRemove,
+                        analyzeRule,
                     });
                     break;
                 case 'custom-selector': {
                     const params = atRule.params.split(/\s/);
```

Comment:
do we want to extract custom-selector to `analyzeAtRule`

Reply:
not in this PR, feature extraction should done in a separate PR